project/Http4sPlugin.scala

```diff
@@ -31,11 +31,11 @@ object Http4sPlugin extends AutoPlugin {
     ThisBuild / http4sApiVersion := (ThisBuild / version).map {
       case VersionNumber(Seq(major, minor, _*), _, _) => (major.toInt, minor.toInt)
     }.value,
-  )
+  ) ++ sbtghactionsSettings
 
   override lazy val projectSettings: Seq[Setting[_]] = Seq(
     scalaVersion := scala_213,
-    crossScalaVersions := Seq(scala_213, scala_212),
+    ThisBuild / crossScalaVersions := Seq(scala_213, scala_212),
```

Comment:
Did you mean adding `crossScalaVersion` to the build scope of each project? If yes then perhaps it might be better to override `def buildSettings` method rather than keep it in `projectSettings` with explicit scoping ([doc](https://www.scala-sbt.org/1.x/docs/Plugins.html#projectSettings+and+buildSettings))

Reply:
Thanks for the link, will do!